runtime/lua/vim/lsp/rpc.lua

```diff
@@ -16,7 +16,7 @@ end
 --- Embeds the given string into a table and correctly computes `Content-Length`.
 ---
 ---@param encoded_message (string)
----@return string containing encoded message and `Content-Length` attribute
+---@return string #string containing encoded message and `Content-Length` attribute
```

Comment:
```suggestion
---@return string string containing encoded message and `Content-Length` attribute
```

Reply:
`@return string # encoded message and `Content-Length` attribute`